hangupsbot/plugins/image/image_memegenerator.py

```diff
@@ -39,8 +45,11 @@ def meme(bot, event, *args):
         if len(results['result']) > 0:
             instanceImageUrl = random.choice(results['result'])['instanceImageUrl']
 
-            image_data = urllib.request.urlopen(instanceImageUrl)
+            request = urllib.request.Request(instanceImageUrl,None,headers) 
+            image_data = urllib.request.urlopen(request)
```

Comment:
What about using `aiohttp` to fetch the image data?

Reply:
Yeah, I think it's possible to do this with `aiohttp` too.